dash/development/_r_components_generation.py

```diff
@@ -200,7 +200,7 @@ def generate_class_string(name, props, project_shortname, prefix):
 
     # pylint: disable=C0301
     default_paramtext += ", ".join(
-        '{}={}'.format(p, p)
+        '{0}={0}'.format(p)
```

Comment:
I think in py27 just {}={} would be equivalent to {0}={1} so the indexes are necessary to have the same behaviour in both 27 and 37?

Reply:
Yes